src/attributes/classes.js

```diff
@@ -78,7 +76,7 @@ jQuery.fn.extend({
 
 				if ( cur ) {
 					j = 0;
-					while ( (clazz = classes[j++]) ) {
+					while ( ( clazz = classes[ j++ ] ) ) {
```

Comment:
Yeah, that is all true, i was asking if we should still do that? Since this is a single expression and if something goes wrong in the loop, we definitely will notice that.